CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
```

Comment:
Somewhere in this file we need to mention the style guides used, along with any other standards we follow.